distributed/cli/tests/test_dask_worker.py

```diff
@@ -19,13 +22,28 @@
 from distributed.deploy.utils import nprocesses_nthreads
 from distributed.metrics import time
 from distributed.utils import parse_ports, sync
-from distributed.utils_test import (
-    gen_cluster,
-    popen,
-    requires_ipv6,
-    terminate_process,
-    wait_for_port,
-)
+from distributed.utils_test import gen_cluster, popen, requires_ipv6
+
+if sys.version_info >= (3, 9):
+    from asyncio import to_thread
+else:
+
+    async def to_thread(*func_args, **kwargs):
```

Comment:
I could see this being useful elsewhere, and it feels a bit odd to inline it in the test file. Perhaps move it to `distributed.utils` instead?